kubernetes.go

```diff
@@ -106,13 +144,31 @@ func (gw *Gateway) RunKubeController(ctx context.Context) error {
 		return err
 	}
 
-	gw.Controller = newKubeController(ctx, kubeClient)
+	gwAPIClient, err := gatewayClient.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	gw.Controller = newKubeController(ctx, kubeClient, gwAPIClient)
 	go gw.Controller.run()
 
 	return nil
 
 }
 
+func checkGatewayCRDs(ctx context.Context, c *gatewayClient.Clientset) bool {
```

Comment:
I was trying to figure out the best way to check for the presence of CRDs in the cluster, and this seems effective.
Definitely going to have to add a modified version of this for VirtualServers on my feature branch.

Reply:
great